src/types.ts

```diff
@@ -105,7 +105,7 @@ export interface PngDecoderOptions {
 
 export interface PngEncoderOptions {
   interlace?: 'null' | 'Adam7';
-  zlib?: DeflateFunctionOptions;
+  zlib?: ZlibOptions;
```

Comment:
Is the shape of the options interface different?

Reply:
Pako options interface looks like this:
 ```ts
   interface DeflateFunctionOptions {
        level?: -1 | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | undefined;
        windowBits?: number | undefined;
        memLevel?: number | undefined;
        strategy?: StrategyValues | undefined;
        dictionary?: any;
        raw?: boolean | undefined;
    }
```
fflate options interface looks like this:

```ts
export interface ZlibOptions extends DeflateOptions {
}

export interface DeflateOptions {
    level?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
    mem?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
    dictionary?: Uint8Array;
}
```

Reply:
@targos Should it be marked as a breaking change?

Reply:
Yes, I think so.